### ISSUE TITLE
3- tier Netweaver Template one currently differs from the instructions

### ISSUE THREAD

**Issue by akn9050** [Microsoft/MCW-SAP-NetWeaver-on-Azure#13]
There is slight changes in template. Instructions of lab guide differ from actual template. Can you please confirm and update

![image](https://user-images.githubusercontent.com/27498287/52552885-c2d68c00-2e07-11e9-9cf2-6847c8c5c2f6.png)

![image](https://user-images.githubusercontent.com/27498287/52552890-c9650380-2e07-11e9-9074-2011fa3fae10.png)

**Comment by fasalzaman** [comment]
Any updates on this?

**Comment by DawnmarieDesJardins** [comment]
@mwasham 
Michael, is this being addressed in the March test/fix?

**Comment by fasalzaman** [comment]
While running the powershel script in Exercise 1 > Task 5 > Step 2, I am getting in to the following error :

```
WARNING: The output object type of this cmdlet will be modified in a future release.
Set-AzureRmNetworkInterface : Different basic sku and standard sku load balancer or public Ip resources in availability set is not allowed /subscriptio
ns/03df3539-4556-46c1-87d1-b5383cd0bb27/resourceGroups/ODL-sap-netweaver-59091-s03RG/providers/Microsoft.Network/loadBalancers/s03-lb-pub-db and /subsc
riptions/03df3539-4556-46c1-87d1-b5383cd0bb27/resourceGroups/ODL-sap-netweaver-59091-s03RG/providers/Microsoft.Network/publicIPAddresses/s03-pip-db-0.
StatusCode: 400
ReasonPhrase: Bad Request
OperationID : '5af4a69d-485f-409b-a3fc-56c30c3ff40c'
At line:41 char:9
+         Set-AzureRmNetworkInterface -NetworkInterface $nic
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Set-AzureRmNetworkInterface], NetworkCloudException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.Network.SetAzureNetworkInterfaceCommand
 
WARNING: The output object type of this cmdlet will be modified in a future release.
Set-AzureRmNetworkInterface : Different basic sku and standard sku load balancer or public Ip resources in availability set is not allowed /subscriptio
ns/03df3539-4556-46c1-87d1-b5383cd0bb27/resourceGroups/ODL-sap-netweaver-59091-s03RG/providers/Microsoft.Network/loadBalancers/s03-lb-pub-db and /subsc
riptions/03df3539-4556-46c1-87d1-b5383cd0bb27/resourceGroups/ODL-sap-netweaver-59091-s03RG/providers/Microsoft.Network/publicIPAddresses/s03-pip-db-1.
StatusCode: 400
ReasonPhrase: Bad Request
OperationID : 'e0382f3b-9a87-4606-ae96-7d49e6ad1ad6'
At line:41 char:9
+         Set-AzureRmNetworkInterface -NetworkInterface $nic
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Set-AzureRmNetworkInterface], NetworkCloudException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.Network.SetAzureNetworkInterfaceCommand
 
WARNING: The output object type of this cmdlet will be modified in a future release.
Set-AzureRmNetworkInterface : IP configuration /subscriptions/03df3539-4556-46c1-87d1-b5383cd0bb27/resourceGroups/ODL-sap-netweaver-59091-s03RG/provide
rs/Microsoft.Network/networkInterfaces/s03-nic-ascs-0/ipConfigurations/ipconfig1 is using the private IP address 10.0.1.8 which is already allocated 
to resource 
```
SKUs of some public IPs and load balancers are different(Basic and Standard).
Can you also look at this?

**Comment by DawnmarieDesJardins** [comment]
Closing issue - addressed in merged PR #17
